### PR TITLE
fix: BRI Reward zero value is approximated

### LINKED ISSUE #744: BRI Reward zero value is approximated
### Context information

application version: 
device: 
browser: 
operating system: 

### Steps to reproduce
go to main page on testnet deployment, see BRI Reward

### What is expected?
0 value

### What is actually happening?
there is `~` symbolizing approximate value
image

### PR DESCRIPTION
**Pull request by janmichek**
## Description
Resolves #744

## Demo
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] fix BRI reward formatting

### CODE REVIEW

File: src/components/KeyblockTable.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@
             class="keyblock-table__value-hash-ellipsed"/>
         </td>
         <th class="keyblock-table__header keyblock-table__column-end">
-          Beneficiary-Reward
+          Beneficiary Reward
```
Comment (janmichek) [inline src/components/KeyblockTable.vue:45]: additional text cleanup to unite with keyblock detail

**Review by michele-franchi — APPROVED** [review]
LGTM

**Review by thepiwo — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
Is this expected on testnet? 
test
est 1

**Comment by thepiwo** [comment]
@Liubov-crypto yes since the Ceres hardfork there is no more BRI

**Comment by Liubov-crypto** [comment]
> @Liubov-crypto yes since the Ceres hardfork there is no more BRI

ok, thank you! 

In this case LGTM

**Pull request merged**